rpa-manager/modules/ROOT/pages/usermanagement-connect.adoc

```diff
@@ -1,6 +1,8 @@
-= Connecting to MuleSoft Composer
+= Connecting to RPA Manager from Salesforce Products
```

Comment:
I think the wording "Connecting to RPA Manager" seems incorrect to me. How about - "Connecting RPA Processes from Salesforce Products"? I feel connecting to RPA manager may be interpreted differently by users.

Reply:
Thank you. That is better.